app/styles/app.css

```diff
@@ -3268,3 +3268,15 @@ Ensure the default browser behavior of the `hidden` attribute.
     line-height: 1.25;
   }
 }
+
+.accordion-title {
```

Comment:
This file is autogenerated if I remember correctly. So if you add something here, it *will* be removed with another build job automatically. It's better to use the Tailwind classes instead since these classes won't "survive" starting the project locally 🙈  for the `accordion-title`, replace that with `mx-4`. For the `hover` class, put `hover:text-vsp-500` on the elements instead 😊